diff -Naur findlib-1.8.1.orig/configure findlib-1.8.1/configure
--- findlib-1.8.1.orig/configure	2019-07-29 21:06:34.000000000 +0900
+++ findlib-1.8.1/configure	2019-12-27 23:34:04.435028631 +0900
@@ -576,7 +576,7 @@
 
 # Generate the META files now.
 
-l="$ldbm dynlink $lgraphics $lnum str threads unix stdlib bigarray ocamldoc $llabltk $lcamlp4 $lobuild $lcomplibs $lbytes $lspacetime"
+l="$ldbm dynlink $lgraphics $lnum str threads unix stdlib bigarray ocamldoc $lcomplibs $lbytes $lspacetime"
 
 for dir in site-lib-src/*; do
     # We do not really know if $dir is a directory.
